fiorella/src/styles/main.scss

```diff
@@ -26,3 +27,7 @@
 /*================ MODULES ================*/
 @import url('modules/site-header.scss');
```

Comment:
Check for the order in which the files should be imported according to the 7-1 Pattern section here: [Sass Guidelines](https://sass-guidelin.es/#architecture)